aioamqp/tests/test_channel.py

```diff
@@ -29,6 +29,16 @@ def test_close(self):
         self.assertFalse(channel.is_open)
 
     @testing.coroutine
+    def test_server_initiated_close(self):
```

Comment:
I'm not really sure that tests ensure we really sent the response for a `channel_close` message